src/entities/profile/model/types.ts

```diff
@@ -3,14 +3,35 @@
  * Use branded type to entity id to
  * don't to be confused with other identifiers
  */
-export type ProfileId = Brand<Id, 'ProfileId'>
+interface IUserFavorite {
+  id: number,
+  name: string
+}
+// export type ProfileId = Brand<Id, 'ProfileId'>
+export type ProfileId = {userId: string};
+
+
 
 export type ProfileDetails = {
-  id: ProfileId
+  id: number
   username: string
   email: string
   firstName?: string
   lastName?: string
   image: Url,
   isEmailVerified: boolean,
-}
+  city: string,
+  is_private: boolean,
+  bio: string,
+  age: number,
+  date_of_birth: string,
+  category_favorite: IUserFavorite[] | []
+} 
+
+// 
+export type ProfileFollowing = {
```

Comment:
Use an interface instead of a type.eslint

https://habr.com/ru/sandbox/186102/ - types vs interfaces